tools/ports/sdl3_image.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2025 The Emscripten Authors.  All rights reserved.
+# Emscripten is available under two separate licenses, the MIT license and the
+# University of Illinois/NCSA Open Source License.  Both these licenses can be
+# found in the LICENSE file.
+
+import os
+
+from typing import Dict, Set
+
+TAG = 'release-3.2.4'
+HASH = '102b8ea30506a1aa0a23196d807e5cdf1ff9efcbcf9db2518a3fffef82f94bda96891129c24a8008a4ec374c9dfc39d5321cf7e573f3e3b10e85b2cfbbc1ad9b'
+
+deps = ['sdl3']
+variants = {
+  'sdl3_image-jpg':    {'SDL3_IMAGE_FORMATS': ["jpg"]},
+  'sdl3_image-png':    {'SDL3_IMAGE_FORMATS': ["png"]},
+  'sdl3_image-jpg-mt': {'SDL3_IMAGE_FORMATS': ["jpg"], 'PTHREADS': 1},
+  'sdl3_image-png-mt': {'SDL3_IMAGE_FORMATS': ["png"], 'PTHREADS': 1},
+}
+
+OPTIONS = {
+  'formats': 'A comma separated list of formats (ex: --use-port=sdl3_image:formats=png,jpg)',
+}
+
+SUPPORTED_FORMATS = {'avif', 'bmp', 'gif', 'jpg', 'jxl', 'lbm', 'pcx', 'png',
+                     'pnm', 'qoi', 'svg', 'tga', 'tif', 'webp', 'xcf', 'xpm', 'xv'}
+
+# user options (from --use-port)
+opts: Dict[str, Set] = {
+  'formats': set(),
+}
+
+
+def needed(settings):
+  return settings.USE_SDL_IMAGE == 3
+
+
+def get_formats(settings):
+  return opts['formats'].union(settings.SDL3_IMAGE_FORMATS)
+
+
+def get_lib_name(settings):
+  formats = '-'.join(sorted(get_formats(settings)))
+
+  libname = 'libSDL3_image'
+  if formats != '':
+    libname += '-' + formats
+  if settings.PTHREADS:
+    libname += '-mt'
+  if settings.SUPPORT_LONGJMP == 'wasm':
+    libname += '-wasm-sjlj'
+  return libname + '.a'
+
+
+def get(ports, settings, shared):
+  ports.fetch_project('sdl3_image', f'https://github.com/libsdl-org/SDL_image/archive/{TAG}.zip', sha512hash=HASH)
```

Comment:
Comparing to the sdl2_image port, I see this in the diff:
```diff
-  ports.fetch_project('sdl2_image', f'https://github.com/libsdl-org/SDL_image/archive/refs/tags/{TAG}.zip', sha512hash=HASH)
+  ports.fetch_project('sdl3_image', f'https://github.com/libsdl-org/SDL_image/archive/{TAG}.zip', sha512hash=HASH)
```
Is `/ref/tags/` not needed?

Reply:
Interesting about github aliases, sounds good.